infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/runtime/LogReplicationClientRouter.java

```diff
@@ -161,11 +166,34 @@ public IClientRouter addClient(IClient client) {
 
     @Override
     public <T> CompletableFuture<T> sendMessageAndGetCompletable(CorfuMsg message) {
-        return sendMessageAndGetCompletable(message, null);
+        throw new UnsupportedOperationException("Deprecated API.");
+    }
```

Comment:
Can this, along with other sendMessage router methods that use `CorfuMsg`, be removed now?

Reply:
Done in the other PR.